utils/_features.py

```diff
@@ -1864,3 +1864,30 @@ def kafkaspan_creationcontext_propagation_with_dd_trace_go(test_object):
         """
         pytest.mark.features(feature_id=258)(test_object)
         return test_object
+
+    @staticmethod
+    def aws_sqs_span_creationcontext_propagation_with_dd_trace_py(test_object):
+        """
+        [AWS-SQS][Span Creation][Context Propagation] with dd-trace-py
+        https://feature-parity.us1.prod.dog/#/?feature=259
+        """
+        pytest.mark.features(feature_id=259)(test_object)
```

Comment:
the feature_id=259 is used by _kafkaspan_creationcontext_propagation_with_dd_trace_js_
You should create the features on FPD and set the ids in the code

Reply:
I've created the new features and assigned the new ids